pdns/dnsdistdist/docs/reference/config.rst

```diff
@@ -205,6 +229,10 @@ Access Control Lists
 
   :param {str} netmasks: A table of CIDR netmask, e.g. ``{"192.0.2.0/24", "2001:DB8:14::/56"}``. Without a subnetmask, only the specific address is allowed.
 
+.. function:: showACL()
+
+  Print a list of all allowed netmasks.
```

Comment:
this should probably say what this ACL is used for...

Reply:
Right, I'll write a few words!